src/configuration/GPG/GnuPG/gpg.conf

```diff
@@ -205,8 +205,7 @@ keyserver hkp://keys.gnupg.net
 # photo-viewer "metamail -q -d -b -c %T -s 'KeyID 0x%k' -f GnuPG"
 
 
-personal-digest-preferences SHA256
-cert-digest-algo SHA256
-default-preference-list SHA512 SHA384 SHA256 SHA224 AES256 AES192 AES CAST5 ZLIB BZIP2 ZIP Uncompressed
-
+personal-digest-preferences SHA512
+cert-digest-algo SHA512
+default-preference-list AES256, CAMELLIA256, AES192, CAMELLIA192, AES, CAMELLIA128, TWOFISH, SHA512, SHA384, SHA256, BZIP2, ZLIB, ZIP 
```

Comment:
usually these options are not seperated by 'commas' but blanks.

furthermore; I'd get rid of CAMELLIA, TWOFISH and AES192 entirely. These are rarely used algorithms, some haven't seen cryptanalysis in ages due to lacking interest or aren't as secure anymore as they used to be at the time they were introduced into GPG.

alternative suggestion:
`default-preference-list SHA512 SHA384 SHA256 AES256 AES ZLIB BZIP2 Uncompressed`